src/dso_api/settings.py

```diff
@@ -176,16 +200,61 @@
     },
     "root": {"level": "INFO", "handlers": ["console"]},
     "loggers": {
-        # "django.db.backends": {"level": "DEBUG", "handlers": ["console"]},
-        "dso_api": {"handlers": ["console"], "level": "DEBUG", "propagate": False},
+        "django": {"handlers": ["console"], "level": DJANGO_LOG_LEVEL, "propagate": False},
+        "dso_api": {"handlers": ["console"], "level": DSO_API_LOG_LEVEL, "propagate": False},
         "dso_api.audit": {
             "handlers": ["structured_console"],
-            "level": "DEBUG",
+            "level": DSO_API_AUDIT_LOG_LEVEL,
             "propagate": False,
         },
     },
 }
 
+if CLOUD_ENV.is_azure():
```

Comment:
`if CLOUD_ENV.startswith("azure")` had ik ook goed genoeg gevonden :-p laat je het wat meer open en zijn de environments niet beperkt tot de enum.

Reply:
In that case I'd rather adjust the definition of `CloudEnv.is_azure()` to:

```python
    def is_azure(self):
        azure = (v for v in CloudEnv.__members__.values() if v.value.startswith("azure"))
        return self in azure
```

Reply:
Limiting the possible environment variables to the enum was done on purpose; to prevent non-sensical values. I don't want devops engineers to set `CLOUD_ENV=aks` and expecting it to work. I want to have the application fail early so that it can be troubleshooted immediately

Reply:
Discussed it offline. I now agree with @vdboor. Ditched the `Enum`